README.md

```diff
@@ -40,9 +40,9 @@ import duxy from 'duxy';
 import duxySuperagent from 'duxy-superagent';
 import request from 'superagent';
 
-const http = duxySuperagent(superagent)();
+const http = duxySuperagent(request)(() => {});
```

Comment:
Without passing a function this throws an exception, this is just a temporary documentation fix, we should patch this in [`duxy-superagent`](https://github.com/producthunt/duxy-superagent/blob/master/src/index.js#L10) and add a conditional